code/global.dm

```diff
@@ -13,6 +13,7 @@ var/global/list/active_diseases          = list()
 var/global/list/med_hud_users            = list() // List of all entities using a medical HUD.
 var/global/list/sec_hud_users            = list() // List of all entities using a security HUD.
 var/global/list/hud_icon_reference       = list()
+var/global/list/listening_objects = list() // List of objects that need to be able to hear, used to avoid recursive searching through contents.
```

Comment:
Not currently implemented anywhere here, but this is part of aurora's more efficient solution for objects that want to hear chat messages. Things like recording devices and intercoms.

Reply:
https://github.com/discordia-space/CEV-Eris/blob/master/code/_helpers/global_lists.dm#L22

Reply:
ahh i see, i'll use that then